mods/ts/rules/defaults.yaml

```diff
@@ -97,6 +97,23 @@
 	EditorTilesetFilter:
 		RequireTilesets: TEMPERAT
 
+^Crate:
+	Tooltip:
+		Name: Crate
+		GenericName: Crate
```

Comment:
Yaml looks fine, but as @obrakmann I don't get the tooltip.
It works when you use `RenderBuilding:` though.